src/technologies/g.json

```diff
@@ -443,16 +443,40 @@
     "scriptSrc": "/wp-content/plugins/gp-premium/.+\\.js(?:\\?ver=(\\d+(?:\\.\\d+)+))?\\;version:\\1",
     "website": "https://docs.generatepress.com/article/installing-gp-premium/"
   },
+  "Genesis blocks": {
+    "cats": [
+      87
+    ],
+    "description": "Premium block library & block editor enhancements from StudioPress.",
+    "icon": "genesis-blocks.png",
+    "requires": "WordPress",
+    "html": "<div[^>]+class=[\"']wp-block-atomic-blocks-*",
+    "dom": {
+      "link[href*='/wp-content/plugins/atomic-blocks']": {
+        "attributes": {
+          "href": "/wp-content/plugins/atomic-blocks/"
```

Comment:
Don't these three line mean the same thing twice?

Reply:
Yeah I had this somewhere else too, not sure why but it doesn't seem to work without the second.

Reply:
Ah maybe the first selects the it applies to and the second is the actual check?

OK, since you tested it let’s go with this then!